项目编译失败

<!-- 

Please try to use English to describe your issue, or at least provide a snippet of English translation.

-->

## Issue Description

Type: *bug report* or *feature request*

### Describe what happened (or what feature you want)
在root目录执行 make命令后编译失败
`[ERROR] Could not find goal 'assembly' in plugin org.apache.maven.plugins:maven-assembly-plugin:3.6.0 among available goals help, single -> [Help 1]`
也尝试网上的一些方法 包括执行 mvn compile assembly:single 等 但还是存在问题
Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.2-beta-5:single (default-cli) on project chaosblade-exec-spi: Error reading assemblies: No assembly descriptors found.
### Describe what you expected to happen


### How to reproduce it (as minimally and precisely as possible)

1. 
2. 
3. 

### Tell us your environment
java 1.8
mac os(m2芯片)
maven: 自带maven(version: 3.6.3)

### Anything else we need to know?

> 

好的，感谢。 我今天尝试了一下， 我把自带的maven 换成了下载的   然后我的环境是 masos(m2) +  idea 2020.3. + maven 3.6.3 目前可以成功make.  老哥通过chaosblade 的那个项目可以直接make吗， 我执行make build_darwin会报错，就只能打包 cli工具。然后根据不同的场景在进行打包，获取yaml文件。
`exec/executor_ssh.go:22:2: missing go.sum entry for module providing package github.com/chaosblade-io/chaosblade-spec-go/log (imported by github.com/chaosblade-io/chaosblade-exec-os/exec/cpu); to add:
        go get github.com/chaosblade-io/chaosblade-exec-os/exec/cpu
` 就类似这种错误，但是我的依赖都导入了

--- a/pom.xml
+++ b/pom.xml
@@ -52,6 +52,8 @@
                 <inherited>false</inherited>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
+                <!-- assembly:assembly in Makefile is not supported since 2.6 -->
+                <version>2.5.5</version>
                 <configuration>
                     <finalName>chaosblade-java-agent</finalName>
                     <descriptors>

